src/components/filters/DescFilter.tsx

```diff
@@ -0,0 +1,109 @@
+import { filterProps } from '@/pages/StatsDashboard'
+import BaseFilter from './BaseFilter'
+import { LucideCalendar, LucideChevronDown, LucideChevronRight, LucideTrash2 } from 'lucide-react'
+import { useEffect, useState } from 'react'
+import {
+  useFloating,
+  offset,
+  flip,
+  shift,
+  autoUpdate,
+  useClick,
+  useDismiss,
+  useRole,
+  useInteractions,
+  FloatingFocusManager,
+} from '@floating-ui/react'
+
+enum DescFilterOption {
+  CONTAINS = 'contiene',
+}
+const possibleDescFilterOptions: DescFilterOption[] = [
+  DescFilterOption.CONTAINS,
+]
+
+const FilterDesc = ({ id, dispatch }: filterProps) => {
+  const [search, setSearch] = useState('')
+  const [selectedDescFilter, setSelectedDescFilter] = useState(DescFilterOption.CONTAINS)
+  const [isDescFilterSelectOpen, setIsDescFilterSelectOpen] = useState(false)
+
+  useEffect(() => {
+    switch (selectedDescFilter) {
+      case DescFilterOption.CONTAINS:
+        dispatch({ type: 'UPDATE_FILTER', payload: { id: id, operation: (incident) => incident.description.includes(search) } })
+        break
+    }
+  }, [search, selectedDescFilter])
+
+  const removeThisFilter = () => {
+    dispatch({ type: 'REMOVE_FILTER', payload: { id: id } })
+  }
+
+  const { refs, floatingStyles, context } = useFloating({
+    open: isDescFilterSelectOpen,
+    onOpenChange: setIsDescFilterSelectOpen,
+    middleware: [offset(10), flip({ fallbackAxisSideDirection: 'end' }), shift()],
+    whileElementsMounted: autoUpdate,
+  })
+
+  const click = useClick(context)
+  const dismiss = useDismiss(context)
+  const role = useRole(context)
+
+  const { getReferenceProps, getFloatingProps } = useInteractions([click, dismiss, role])
+
+  return (
+    <BaseFilter
+      icon={<LucideCalendar />}
+      text={
+        `La descripción contiene «${search}»`
+      }
+    >
+      <button onClick={removeThisFilter} className="absolute right-2 top-1 h-4 w-4 text-red-600" title="Eliminar Filtro">
+        <LucideTrash2 size={20} />
+      </button>
+
+      {/* Dropdown to select the filter option */}
+      <div
+        onClick={() => setIsDescFilterSelectOpen(true)}
+        className="flex cursor-pointer items-center text-sm"
+        ref={refs.setReference}
+        {...getReferenceProps()}
+      >
+        <p className="flex flex-row items-center text-gray-600">
+          Fecha {selectedDescFilter}
```

Comment:
This isn't the date filter anymore, so you should remove references to that.

```suggestion
          La descripción {selectedDescFilter}
```